src/System.Numerics.Vectors/src/System/Numerics/Vector2_Intrinsics.cs

```diff
@@ -209,7 +211,6 @@ public static Vector2 SquareRoot(Vector2 value)
         /// <param name="left">The scalar value.</param>
         /// <param name="right">The source vector.</param>
         /// <returns>The scaled vector.</returns>
-        [Intrinsic]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static Vector2 operator *(float left, Vector2 right)
```

Comment:
The runtime only matches against `TYP_STRUCT, TYP_STRUCT`: https://github.com/dotnet/coreclr/blob/master/src/jit/simdintrinsiclist.h#L91

Reply:
> it might be handled by .NET Native

ARM seems to have instructions can directly support this form, so you may need to care about that.